docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -47,6 +47,17 @@ In virtual environments, high CPU steal indicates that the virtual machines migh
 
 *Further Reading*: https://docs.couchbase.com/server/current/learn/buckets-memory-and-storage/storage.html[Storage]
 
+[#CB90044]
+=== Babysitter Managed Process Crash (CB90043)
+
+*Background*: Babysitter is part of Couchbase Server's cluster manager which is responsible for maintaining a variety of Couchbase Server processes.
+
+*Condition*: A process managed by babysitter crashes.
+
+*Remediation*: Contact Couchbase Technical Support.
```

Comment:
Do we need to go straight to this? Things can occasionally crash for transient reasons (e.g. OOM kills) and carry on working fine. Perhaps "If the process keeps crashing repeatedly or you notice disruption to your cluster..."